src/driver/aurora-mysql/AuroraMysqlQueryRunner.ts

```diff
@@ -1915,7 +1915,10 @@ export class AuroraMysqlQueryRunner
                 if (!isAnotherTransactionActive) {
                     await this.rollbackTransaction()
                 }
-            } catch (rollbackError) {}
+            } catch {
+                // ignore
+            }
```

Comment:
nit: it may be nice to provide context on why we want to ignore these errors and/or what we are ignoring. like "ignore rollback errors"